### PR TITLE
proxy_init uses proxy hub/tag, fix mixer health yaml generation

### PR DESCRIPTION
**Pull request by rkpagadala**

### CODE REVIEW

**Comment by rkpagadala** [comment]
followup of #4179 #4181

File: install/updateVersion.sh [hunk 1]
```diff
@@ -287,19 +287,19 @@ function update_istio_install() {
   execute_sed "s|image: {MIXER_HUB}/\(.*\):{MIXER_TAG}|image: ${MIXER_HUB}/\1:${MIXER_TAG}|" istio-mixer-with-health-check.yaml.tmpl
   execute_sed "s|image: {MIXER_HUB}/\(.*\):{MIXER_TAG}|image: ${MIXER_HUB}/\1:${MIXER_TAG}|" istio-mixer-validator.yaml.tmpl
   execute_sed "s|image: {PROXY_HUB}/{PROXY_IMAGE}:{PROXY_TAG}|image: ${PROXY_HUB}/${PROXY_IMAGE}:${PROXY_TAG}|" istio-mixer.yaml.tmpl
+  execute_sed "s|image: {PROXY_HUB}/{PROXY_IMAGE}:{PROXY_TAG}|image: ${PROXY_HUB}/${PROXY_IMAGE}:${PROXY_TAG}|" istio-mixer-with-health-check.yaml.tmpl
```
Comment (jmuk) [inline install/updateVersion.sh]: thanks for fixing this!

**Comment by mattdelco** [comment]
Looks like I overlooked the consul vs eureka and failed to mention this file:
<pre>
bookinfo/consul/templates/bookinfo.sidecars.yaml.tmpl
</pre>
and once that's updated I think update_istio_install_docker() in updateVersions.sh no longer needs to call sed on bookinfo.sidecars.yaml.tmpl for PILOT_HUB/PILOT_TAG.

**Comment by mattdelco** [comment]
/lgtm
thanks

**Pull request merged**